test/Rize/Uri/Node/ParserTest.php

```diff
@@ -85,4 +86,45 @@ public function testParseTemplate()
 
         $this->assertEquals($expected, $actual);
     }
-}
\ No newline at end of file
+
+    public function testParseTemplateWithLiteral()
+    {
+        // will pass
+        $uri = new UriTemplate('http://www.example.com/v1/company/', []);
```

Comment:
Seems like `[]` makes tests in travis-ci failed in php 5.3 (even though it's no longer supported). Maybe you can remove this or change to `array()`)